websites/A/Aodesu/metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "892219412302671913",
+		"name": "Ao"
+	},
+	"service": "Aodesu",
+	"description": {
+		"en": "Aodesu is a streaming website where you can watch anime with Spanish subtitle or Dubs in Spanish.",
+		"es": "Aodesu es un sitio web de streaming donde podrás ver anime subtitulado al español o doblados al español."
+	},
+	"url": [
+		"aodesu.com",
+		"www.aodesu.com"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/CyVxys5.jpg",
+	"thumbnail": "https://i.imgur.com/3utIKC9.png",
+	"color": "#4169e1",
+	"category": "anime",
+	"tags": [
+		"aodesu",
+		"streaming",
+		"videos"
+	],
+	"iframe": true,
+	"iFrameRegExp": ".*"
```

Comment:
@Bas950

This should work for all iframes that are currently used on the site. See proof below.
```suggestion
	"iFrameRegExp": "(yourupload[.]com)|(guccihide[.]com)|(streamtape[.]com)|(vidcache[.]net)|(burstcloud[.]co)|((.*[.])?google[.]com)"
```

## Screenshots
<details>
<summary> Proof showing the request changed is working as expected </summary>

<img src="https://user-images.githubusercontent.com/42322979/235454178-4b0f0f29-0f34-4504-8888-edf9babe686b.png">
<img src="https://user-images.githubusercontent.com/42322979/235454181-f97beec5-2b24-494c-8046-b12ed2c9fa95.png"><img src="https://user-images.githubusercontent.com/42322979/235454182-40dc773b-67a9-4818-9d8d-ff37e2d0336f.png">


</details>

Reply:
I got it, thanks